src/main/java/org/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -74,6 +80,35 @@ public String getName() {
         }
     }
 
+    public String getHeading() {
+        return heading;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void confirmRecommendations(String status) {
```

Comment:
This is not used? (or at least I couldn't find a reference to it)

Reply:
Yep, sorry that's not used. We've removed it now. Thanks!